### DISCUSSION TITLE
Submit Feed - POST_FLAT_FILE_INVLOADER_DATA

### DISCUSSION THREAD

**Discussion by Lymosx** [abuzuhri/Amazon-SP-API-CSharp#396] [Q&A]
Hello, I'm getting an error trying to get into the list.

Call Flat File -
![1](https://user-images.githubusercontent.com/55663430/200961967-8150d76c-3942-419e-9ee5-2495a71997fa.png)
Post File Data - 
![2](https://user-images.githubusercontent.com/55663430/200961985-83a4befd-da69-4afa-9f22-7fa49d2b7760.png)

How can I fix it ?

**Comment by RenzoF** [comment]
Hi,

Try passing the file path rather than the file contents on the SubmitFeed method.
So ... Feed.SubmitFeed("C:\users\test.txt", FeedType...

**Reply by Lymosx** [reply]
There is no problem when I upload it as a Spreadsheet, but I encounter an error when I upload it using the method you mentioned.

Code :
![code](https://user-images.githubusercontent.com/55663430/201211229-30b08025-8ad6-47a8-a4e2-0013bf39cbce.png)
Error:
![complete](https://user-images.githubusercontent.com/55663430/201211486-9093d180-cb6c-4026-991f-3a09835b8084.png)

Error Message:
`90053	Fatal	The file has an incorrect template type. To correct this error, download the correct template from seller Help.`

Spreadsheet:
![err](https://user-images.githubusercontent.com/55663430/201211275-a2f40061-e9b1-4f9e-bf3d-369fa7959dca.png)

Text File:
`TemplateType=Offer	Version=2020.000					
sku	price	quantity	product-id	product-id-type	condition-type	condition-note
MD-xxxxxxxxx	62,45	3	B0xxxxxxx	ASIN	NEW	In original package
`

**Reply by RenzoF** [reply]
Right, this is a different issue, at least your file is being posted now.
For your test file try save as tsv from your (working) xlsx template sheet and use that instead
